src/main/scala/io/iohk/ethereum/db/components/StoragesComponent.scala

```diff
@@ -19,7 +19,7 @@ trait StoragesComponent {
 
     val mptNodeStorage: MptNodeStorage
 
-    val nodeStorage: NodeStorage //FIXME This storage is similar to MPTNodesStorage but
+    val nodeStorage: NodeStorage //FIXME This storage is similar to MPTNodesStorage, should we keep only one? [EC-147]
```

Comment:
they are pointing underneath to the same namespace `Namespaces.NodeNamespace`, but they are used for 2 different purpose, one is for fast sync it operates on objects defined for wire protocol 
second is for our internal implementation of mpt it is used when executing transactions and it uses objects defined for mpt